docs/api-inference/getting-started.md

```diff
@@ -0,0 +1,78 @@
+# Getting Started
+
+The Serverless Inference API allows you to easily do inference on a wide range of models and tasks. You can do requests with your favorite tools (Python, cURL, etc). We also provide a Python SDK (`huggingface_hub`) to make it even easier.
+
+We'll do a minimal example using a [sentiment classification model](https://huggingface.co/cardiffnlp/twitter-roberta-base-sentiment-latest). Please visit task-specific parameters and further documentation in our [API Reference](./parameters.md).
+
+## Getting a Token
+
+Using the Serverless Inference API requires passing a user token in the request headers. You can get a token by signing up on the Hugging Face website and then going to the [tokens page](https://huggingface.co/settings/tokens). We recommend creating a `Fine-grained` token with the scope to `Make calls to the serverless Inference API`.
+
+TODO: add screenshot
+For more details about user tokens, check out [this guide](https://huggingface.co/docs/hub/en/security-tokens).
+
+## cURL
+
+```bash
+curl https://api-inference.huggingface.co/models/cardiffnlp/twitter-roberta-base-sentiment-latest \
+    -X POST \
+    -d '{"inputs": "Today is a nice day"}' \
+    -H "Authorization: Bearer hf_***" \
+    -H "Content-Type: application/json"
```

Comment:
text-generation example would be better no?

Reply:
meta-llama-3.1-8B-Instruct

Reply:
done in https://github.com/huggingface/hub-docs/pull/1408